engine/app/components/citizens_advice_components/targeted_content.rb

```diff
@@ -4,6 +4,20 @@ module CitizensAdviceComponents
   class TargetedContent < Base
     attr_reader :id, :type
 
+    # @example
+    #   <%= render CitizensAdviceComponents::TargetedContent.new(
+    #     id: "example",
+    #     title: "Example targeted content",
+    #     type: :adviser
+    #   ) do %>
+    #     Example content
+    #   <% end %>
+    #
+    # @param title [String] Title
+    # @param id [String] A HTML-friendly ID
+    # @param type [Symbol] Type, either :public or :adviser with a fallback of :public
+    # @param heading_level [Integer] Optional, sets the heading level of the title. Defaults to `2`
```

Comment:
If we specify these defaults in the signature, yard will pick the up automatically.